bin/runtime/src/lib.rs

```diff
@@ -707,6 +711,41 @@ impl pallet_utility::Config for Runtime {
     type PalletsOrigin = OriginCaller;
 }
 
+parameter_types! {
+    // The following weights are pulled out of thin air. A separate task to adjust them is already in JIRA.
+    pub const DepositPerItem: Balance = TOKEN / 100;  //  10 milli-AZERO per item
+    pub const DepositPerByte: Balance = TOKEN / 100_000;      // ~10 milli-AZERO per kB storage
+    // The lazy deletion runs inside on_initialize.
```

Comment:
since you have already provided the real values for `DepositPer_`, I would encourage you to add similar estimations for the two parameters below